downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -48,6 +48,16 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
 
 ## Ballerina library updates
 
+### New features
+
+#### `data.jsondata` module
+
+- Introduced the `data.jsondata` package to provide a set of APIs to work with JSON data.
```

Comment:
```suggestion
- Introduced the `data.jsondata` package to provide a set of APIs for type conversions and data projections.
```
wdyt?

Reply:
It should be as below right?

Introduced the `data.jsondata` package to provide a set of APIs for JSON type conversions, data projections and json navigation.

Reply:
Updated https://github.com/ballerina-platform/ballerina-dev-website/pull/8877/commits/542967ef6683be36f7d7c53bb289ca4adb747c9b